test/FunctionalTests/Microsoft.OData.Client.Tests/TestHttpWebRequestMessage.cs

```diff
@@ -0,0 +1,60 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ClientEdmModelTests.cs" company="Microsoft">

```

Comment:
Should this be TestHttpWebRequestMessage.cs

Reply:
@wachugamaina Fixed